en/Plugins/Search.md

```diff
@@ -7,7 +7,7 @@ By default, you can find Search in the left sidebar (magnifying glass icon). You
 
 ## Search terms
 
-A search term is the word or phrase that you enter in the search field. Learning how to write search terms effectively can help you quickly find what you're looking for, even in large vaults.
+A search term is the word or phrase that you enter in the search field. Learning how to write search terms effectively can help you quickly find what you're looking for, even in large vaults. By default, the search is performed only over all markdown notes and canvases (both their filenames and their content).
```

Comment:
```suggestion
A search term is the word or phrase that you enter in the search field. Learning how to write search terms effectively can help you quickly find what you're looking for, even in large vaults. By default, Obsidian only searches notes and canvases, including their filenames.
```

According to the [style guide](https://help.obsidian.md/Contributing+to+Obsidian/Style+guide#Notes%2C+files%2C+and+folders), "note" implies Markdown.